### PR TITLE
ARC-2322 adding logs to help debug a couple of dos tickets

### PR DESCRIPTION
**Pull request by rachellerathbone**
**What's in this PR?**
Additional, temporary logs.

**Why**
To try and pin down why some customers are missing data.

**Added feature flags**
temp-logs-for-dos-tickets

**Affected issues**  
ARC-2322

**How has this been tested?**  
Locally

**Whats Next?**
Find the reason why customers aren't seeing data :blob_pray: then circle back and clean these logs up.

### CODE REVIEW

File: src/config/feature-flags.ts [hunk 1]
```diff
@@ -29,6 +29,7 @@ export enum BooleanFlags {
 	ENABLE_GENERIC_CONTAINERS = "enable-generic-containers",
 	ENABLE_GITHUB_SECURITY_IN_JIRA = "enable-github-security-in-jira",
 	DELETE_MESSAGE_ON_BACKFILL_WHEN_OTHERS_WORKING_ON_IT = "delete-message-on-backfill-when-others-working-on-it",
+	TEMP_LOGS_FOR_DOS_TICKETS = "temp-logs-for-dos-tickets",
```
Comment (gxueatlassian) [inline src/config/feature-flags.ts:32]: Can you use `VERBOSE_LOGGING` instead?

Comment (gxueatlassian) [reply src/config/feature-flags.ts:32]: Or does it make sense to use `VERBOSE_LOGGING`?

Comment (rachellerathbone) [reply src/config/feature-flags.ts:32]: I can. Just wondering if it's going to add any extra value though?

Comment (rachellerathbone) [reply src/config/feature-flags.ts:32]: Actually, I just checked that feature flag and it looks like it's already on for several customers. I really want to limit this to the 2 customers who I'm helping out. Starting with one that has given permission for me to log this additional data (waiting for the other one to get back to me).

File: src/sqs/backfill-error-handler.ts [hunk 2]
```diff
@@ -19,10 +20,16 @@ const handleTaskError = async (sendSQSBackfillMessage: (message, delaySec, logge
 		receiveCount: context.receiveCount,
 		lastAttempt: context.lastAttempt
 	});
-	log.info("Handling error task");
+
+	const logAdditionalData = await booleanFlag(BooleanFlags.VERBOSE_LOGGING, context.payload.installationId.toString());
```
Comment (rachellerathbone) [inline src/sqs/backfill-error-handler.ts:24]: I don't think there's any way to access the JH here so thought I'd give this a crack instead?

**Review by gxueatlassian — APPROVED** [review]

**Pull request merged**